### PR TITLE
OR-2480 Add article, vote and chat modules

### PR DESCRIPTION
**Pull request by chamerling**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -23,16 +23,23 @@ RUN git clone --depth=1 https://github.com/linagora/openpaas-esn.git .
 
 COPY ./data/populate/index.js ./fixtures/populate/index.js
 COPY ./data/populate/data/populate-objects.js ./fixtures/populate/data/populate-objects.js
+COPY ./data/fixtures/esn-config/data/homePage.js ./fixtures/esn-config/data/homePage.js
+COPY ./data/config/default.json ./config/default.production.json
+COPY ./data/config/default.json ./config/default.development.json
```
Comment (sngo25) [inline Dockerfile:28]: is development needed?

File: data/config/default.json [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    }
+  },
+  "auth": {
+    "strategies": ["local", "mongo", "bearer"]
+  },
+  "modules": [
+    "linagora.esn.account",
+    "linagora.esn.appstore",
+    "linagora.esn.calendar",
+    "linagora.esn.contact",
```
Comment (sngo25) [inline data/config/default.json]: we must disable other modules and keep only chat and article

File: Dockerfile (hunk 1 above)
Comment (chamerling) [reply Dockerfile:28]: just in case...

File: data/config/default.json (hunk 2 above)
Comment (chamerling) [reply data/config/default.json]: yes I know, but when I tried I add some errors. Will check locally and update

File: Dockerfile (hunk 1 above)
Comment (sngo25) [reply Dockerfile:28]: OK

File: data/config/default.json (hunk 2 above)
Comment (sngo25) [reply data/config/default.json]: OK

**Review by sngo25 — APPROVED** [review]

**Pull request merged**